order_test.go

```diff
@@ -696,3 +697,384 @@ func TestOrderCancelFulfillment(t *testing.T) {
 
 	FulfillmentTests(t, *returnedFulfillment)
 }
+
+// TestLineItemUnmarshalJSON tests unmarsalling a LineItem from json
+func TestLineItemUnmarshalJSON(t *testing.T) {
+	setup()
+	defer teardown()
+
+	actual := LineItem{}
+
+	err := actual.UnmarshalJSON(loadFixture("orderlineitems/valid.json"))
+	if err != nil {
+		t.Errorf("LineItem.UnmarshalJSON returned error: %v", err)
+	}
+
+	expected := validLineItem()
+
+	testLineItem(t, expected, actual)
+}
+
+// TestLineItemUnmarshalJSON0- tests unmarsalling a LineItem from invalid
```

Comment:
Uber minor, godoc func name doesn't match

Reply:
Good catch, updated, this one and the following ones that didn't match.